### PR TITLE
Added Jump_Search in C++,python,JavaScript

### PR DESCRIPTION
**Pull request by khushal87**
Refer Issue #697

### CODE REVIEW

File: Jump_Search/Jump_Search.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// Cpp program to implement Jump Search 
+#include <bits/stdc++.h> 
+using namespace std; 
+
+int jumpSearch(int arr[], int x, int n)
```
Comment (jainaman224) [inline Jump_Search/Jump_Search.cpp]: Use more descriptive name than x and n.

Comment (khushal87) [reply Jump_Search/Jump_Search.cpp]: Done

**Comment by jainaman224** [comment]
@khushal87 Please squash your commits.

**Comment by khushal87** [comment]
@jainaman224  I tried dont it but getting an error that the commits are not staged

**Comment by jainaman224** [comment]
@khushal87 How are you doing?
You can also ask for help in slack public channel.

File: Jump_Search/Jump_Search.cpp [hunk 2]
```diff
@@ -0,0 +1,58 @@
+// Cpp program to implement Jump Search 
+#include <bits/stdc++.h> 
+using namespace std; 
+
+int jumpSearch(int arr[], int element, int number_of_element) 
+{ 
+	// Finding block size to be jumped
```
Comment (jainaman224) [inline Jump_Search/Jump_Search.cpp]: Fix indentation.

File: Jump_Search/Jump_Search.cpp [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	int index = jumpSearch(arr, element, number_of_eleemnt); 
+
+	// Print the index where 'element' is located 
+	cout << "\nNumber " << element << " is at index " << index; 
+	return 0; 
+} 
+
+//OUTPUT => Number 55 is at index 10
+//Time Complexity : O(√n)
+
```
Comment (jainaman224) [inline Jump_Search/Jump_Search.cpp]: Use only 1 new line at the end of file.

File: Jump_Search/Jump_Search.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Python3 code for Jump Search 
+import math 
+
+def jumpSearch( arr , element , number_of_elements ): 
+	
+	# Finding block size to be jumped 
+	step = math.sqrt(number_of_elements)
```
Comment (jainaman224) [inline Jump_Search/Jump_Search.py]: If we type cast here. Is there still need to do so at later stages?

File: Jump_Search/Jump_Search.cpp (hunk 2 above)
Comment (khushal87) [reply Jump_Search/Jump_Search.cpp]: What will i do here ?? Its indented i guess

File: Jump_Search/Jump_Search.cpp (hunk 3 above)
Comment (khushal87) [reply Jump_Search/Jump_Search.cpp]: done

File: Jump_Search/Jump_Search.py (hunk 4 above)
Comment (khushal87) [reply Jump_Search/Jump_Search.py]: Yes here it was for step and down there it was for step =step + sqrt of number of elements

File: Jump_Search/Jump_Search.cpp (hunk 2 above)
Comment (jainaman224) [reply Jump_Search/Jump_Search.cpp]: It is indented with 8 spaces. Change it to 4 spaces.

File: Jump_Search/Jump_Search.py (hunk 4 above)
Comment (jainaman224) [reply Jump_Search/Jump_Search.py]: Ok 👍

File: Jump_Search/Jump_Search.cpp (hunk 2 above)
Comment (khushal87) [reply Jump_Search/Jump_Search.cpp]: done

File: Jump_Search/Jump_Search.cpp [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+using namespace std; 
+
+int jumpSearch(int arr[], int element, int number_of_element) 
+{ 
+    // Finding block size to be jumped 
+    int step = sqrt(number_of_element); 
+
+    // Finding the block where element is 
+    // present (if it is present) 
+	int prev = 0;
```
Comment (jainaman224) [inline Jump_Search/Jump_Search.cpp]: Please fix indentation in all files.

Comment (khushal87) [reply Jump_Search/Jump_Search.cpp]: done

File: Jump_Search/Jump_Search.cpp [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+int jumpSearch(int arr[], int element, int number_of_element) 
+{ 
+    // Finding block size to be jumped 
+    int step = sqrt(number_of_element); 
+
+    // Finding the block where element is 
+    // present (if it is present) 
+    int prev = 0; 
+	while (arr[min(step, number_of_element)-1] < x)
```
Comment (jainaman224) [inline Jump_Search/Jump_Search.cpp]: Fix indentation in all files.

**Comment by khushal87** [comment]
All changes are done

**Comment by jainaman224** [comment]
Squash commits and it is ready to be merged.

**Pull request closed without merging**